toast-tk-web-agent/src/main/java/io/toast/tk/agent/web/RestRecorderService.java

```diff
@@ -1,69 +1,86 @@
 package io.toast.tk.agent.web;
 
 
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.http.HttpServer;
+import io.vertx.core.http.HttpServerOptions;
+import io.vertx.core.net.JksOptions;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.handler.BodyHandler;
+import io.vertx.ext.web.handler.ErrorHandler;
 import org.apache.commons.lang3.SystemUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.vertx.java.core.Handler;
-import org.vertx.java.core.http.HttpServerRequest;
-import org.vertx.java.core.http.RouteMatcher;
-import org.vertx.java.platform.Verticle;
 
 import com.google.inject.Guice;
 import com.google.inject.Injector;
 
-import io.toast.tk.agent.config.WebConfig;
+import io.toast.tk.agent.config.AgentConfig;
 import io.toast.tk.agent.guice.WebAgentModule;
 import io.toast.tk.agent.ui.MainApp;
 import io.toast.tk.agent.ui.NotificationManager;
 import io.toast.tk.agent.web.rest.PingHandler;
 import io.toast.tk.agent.web.rest.RecordHandler;
 import io.toast.tk.agent.web.rest.StopHandler;
 
-public class RestRecorderService extends Verticle {
+
+public class RestRecorderService extends AbstractVerticle {
 
 	private static final Logger LOG = LogManager.getLogger(RestRecorderService.class);
 
 	private Injector injector;
-
 	private MainApp app;
 	 
 	@Override
-	public void start() {
+	public void start(Future<Void> fut) {
 		//LOG.info("Starting..");

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)